groups/alteryx-live/data.tf

```diff
@@ -31,6 +31,10 @@ data "vault_generic_secret" "azure_dc_cidrs" {
   path = "applications/${var.aws_account}-${var.aws_region}/${var.application}/azure_dc"
 }
 
+data "vault_generic_secret" "concourse_cidrs" {
```

Comment:
Can we look these up dynamically instead please. That will ease the transition once we start to migrate to the new Vault instance